packages/css/src/input.css

```diff
@@ -51,6 +56,20 @@
   @composes ds-print-preserve from './base.css';
   @composes ds-focus from './base.css';
 
+  &:not(:focus-visible):not(:disabled):not([readonly]):hover {
+    outline-color: var(--dsc-input-outline-color--hover);
+    outline-width: var(--dsc-input-outline-width--hover);
+    outline-style: var(--dsc-input-outline-style--hover);
+
+    &:is(:checked, :indeterminate:where([type='checkbox'])):not([aria-invalid='true']) {
+      outline-color: var(--dsc-input-outline-color-checkbox-radio-switch--hover);
```

Comment:
Earlier, we have used `--toggle` as suffix for properties affecting `radio/checkbox/switch` (i.e. `--dsc-input-border-width--toggle`) since `toggle` is somewhat a "state" of the `input`. Should we do the same here? :)